src/RZA1/intc/devdrv_intc.h

```diff
@@ -32,6 +32,7 @@
 /******************************************************************************
 Includes   <System Includes> , "Project Includes"
 ******************************************************************************/
+#include <stdint.h>
```

Comment:
Technically this should probably be `r_typedefs` (which also defines the integer types which is its own can of worms). I'm not opposed to using this as an excuse to move to the standard headers though.

Reply:
Stdint is already in use in the project, we might as well commit to it

Reply:
Yeah I think I'd rather break away from using r_typedefs.h towards standard C headers.